Library/Formula/cppcms.rb

```diff
@@ -0,0 +1,85 @@
+class Cppcms < Formula
+  desc "A Free High Performance Web Development Framework"
+  homepage "http://cppcms.com/wikipp/en/page/main"
+  url "https://downloads.sourceforge.net/project/cppcms/cppcms/1.0.5/cppcms-1.0.5.tar.bz2"
+  sha256 "84b685977bca97c3e997497f227bd5906adb80555066d811a7046b01c2f51865"
+
+  depends_on "cmake" => :build
+  depends_on "pcre"
+  depends_on "openssl"
+  depends_on :macos => :mavericks
```

Comment:
> Did you try a fails_with or needs :cxx11?
Would you please tell me how to use these options in formulas? Or where is the documents/examples of these options? Thanks.